web/src/main/java/org/cbioportal/genome_nexus/web/mixin/my_variant_info_mixin/MyVariantInfoMixin.java

```diff
@@ -14,6 +14,9 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class MyVariantInfoMixin
 {
+    @ApiModelProperty(value = "variant", required = false)
```

Comment:
Apparently all the `ApiModelProperty` tags were initially used with the wrong intention. This does not do the value mapping, this is for user friendly API documentation. `JsonProperty` is the tag to use for object mapping.